src/dotnet/Notification.Service/Notifications.Backend.cs

```diff
@@ -157,6 +163,18 @@ public virtual async Task<ImmutableArray<Symbol>> ListSubscriberIds(string chatI
 
         }
 
+        async Task<string> AuthorNameResolver(string authorId)
```

Comment:
why such naming like it's delegate or class?

Reply:
What do you suggest then